Runtimes/Core/core/CMakeLists.txt

```diff
@@ -239,6 +239,11 @@ add_library(swiftCore
   "${CMAKE_CURRENT_BINARY_DIR}/UnsafeRawBufferPointer.swift"
   "${CMAKE_CURRENT_BINARY_DIR}/Tuple.swift")
 
+# https://github.com/swiftlang/swift/issues/77705 - Freestanding and Linux/Android builds both have failures to resolve.
+if(NOT LINUX AND NOT ANDROID)
```

Comment:
@compnerd Would it be preferred to use a cmake flag here which can be toggled on/off? Or will this suffice?